### PR TITLE
WARP 602 - Adds foundation content and footer content

### PR DESCRIPTION
**Pull request by dtalakoubi404**

### CODE REVIEW

File: docs/foundations/tokens/intro/index.md [hunk 1]
```diff
@@ -26,26 +22,12 @@ Add token support status on brands: https://nmp-jira.atlassian.net/browse/WARP-5
...
-
-If you are building a component that doesn’t exist in WARP you can still use semantic tokens and get support for dark mode and all other SMP brands for free.
-
-### Interactive states
-Most of the semantic colour tokens also have hover and active states. For the sake of not causing a cognitive overload, those are not visible in this list.
-
-
-
-
-Random test
```
Comment (Skadefryd) [inline docs/foundations/tokens/intro/index.md:48]: Is this supposed to be here? :)

File: docs/help/faq/index.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
 # FAQ
 To answer the most frequently asked questions the following list is constantly updated. Please read through before using the support and communication channels for questions.
+
+## General
+
+**What are tokens?** <br>
```
Comment (Skadefryd) [inline docs/help/faq/index.md:6]: MAybe use headings insteaad of strong , that will help accessibility I think.

**Review by Skadefryd — APPROVED** [review]
A few comments , but nothing breaking :)

File: docs/foundations/tokens/intro/index.md (hunk 1 above)
Comment (dtalakoubi404) [reply docs/foundations/tokens/intro/index.md:48]: No - let's remove this! :)

File: docs/help/faq/index.md (hunk 2 above)
Comment (dtalakoubi404) [reply docs/help/faq/index.md:6]: Sure, we can do that!

**Pull request merged**